### ISSUE TITLE
how to add multiple tests on already authenticated session.

### ISSUE THREAD

**Issue by poojak1** [DevExpress/testcafe#5310]
Hi 
How can I add multiple tests on already authenticated session.
Currently, in every test my application login and logout. I want to login only once and execute the multiple test on authenticated session.

How can I achieve that ?

**Comment by AlexSkorkin** [comment]
Thank you for your inquiry. This issue looks like a question that would be best [asked on StackOverflow](https://stackoverflow.com/questions/ask?tags=testcafe) - an amazing platform for users to ask and answer questions. We try to keep the GitHub issues tracker for bugs and feature requests only (see also: [Contributing](https://github.com/DevExpress/testcafe#contributing)). If you think that this issue is a bug, feel free to [open a new issue](https://github.com/DevExpress/testcafe/issues/new?template=bug-report.md), but please make sure to follow the bug template. Thank you in advance.
